### PR TITLE
Milestone2 : components

### PR DESCRIPTION
**Pull request by abdessalem1998**
In this pull request i need to review my code for making my first react compontent that holds the main app which is the calculator :

- Use class based components.
- Develop just the UI of the calculator.
- import class based components and use it in your main component (App.js).
- Test it in the browser

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (hiromataba) [inline src/index.js]: - [x] In your `index.js` you  should render your `<App/>` component and then your `<Calculator/>` should be rendered inside your `<App/>` component, Kindly fix that to make your code look more awesome and maintanable

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @abdessalem1998 :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight :tada: :+1: 
- Your calculator is responsive :heavy_check_mark: 
- No linter errors :+1: 
- Nice PR title and description :+1: 

## Required Changes ♻️

- [x] Please make your  App.js component a class based component.

image

- [x] Please remove the images and icons in your public directory because you are not going to use them

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] [OPTIONAL] Please add a command in your available script section in your README file to install npm packages, `npm install`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @abdessalem1998, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [x] Please remove all the boilerplate that got created by the CRA (texts, images, and styles) in your `public `directory as the previous reviewer requested because you are not going to use them
- [ ] [OPTIONAL] Also kindly add the `npm start` command to install packages in the `Getting Started` section of your Readme.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @abdessalem1998  🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**